source/frontend/src/features/mapSideBar/research/ResearchView.tsx

```diff
@@ -79,7 +79,7 @@ const ResearchView: React.FunctionComponent<IResearchViewProps> = props => {
     return (
       <MapSideBarLayout
         title={props.isEditing ? 'Update Research File' : 'Research File'}
-        icon={<MdTopic title="User Profile" size="2.5rem" className="mr-2" />}
+        icon={<MdTopic title="User Profile" size="2.5rem" />}
```

Comment:
nit: all other icons are sized at 2.6 rem - would it make sense to adjust this one too?